crypto/provider_core.c

```diff
@@ -676,44 +678,70 @@ static int provider_init(OSSL_PROVIDER *prov, int flag_lock)
     return ok;
 }
 
+/*
+ * Activate a provider.
+ * Return -1 on failure and the number of activations on success
```

Comment:
The number of *de*activations?

Reply:
Also: s/Activate a provider/Deactivate a provider/

Reply:
Fixed

Reply:
I've changed the wording to _the activation count on success_.  This is clearer I think.